docs/helm-chart-checks.md

```diff
@@ -131,18 +131,34 @@ This section provides help on the basic usage of Helm chart checks with the podm
 
 ### Cluster Config
 
-Configuration of the `chart-testing` check can be achieved through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
+Configuration of the `chart-testing` check can be achieved in one of two ways: 
+
+1. Through the `--set` command line option, which in addition of configuring `chart-verifier` itself, can set check's configuration values.
```

Comment:
Question: In this sentence: "Through the `--set` command line option, which in addition of configuring `chart-verifier` itself," I am not clear on:

-  what we mean by: "which in addition of configuring `chart-verifier` itself"?
- Is configuring `chart-verifier` a prerequisite to this step?
- What you mean when you say - "itself"?
- Is the `chart-verifier` automatic and done by the operator?
- What value add does it bring when you mention about the `chart-verifier` configuration in the step for `chart-testing` check configuration? Can we just avoid this? If yes, then I suggest:
"From the `--set` option in the Red Hat Helm chart-verifier CLI, set the configuration values of the `chart-testing` check as follows:"

Reply:
I changed this to just be: 
* Option 1: Through the `--set` command line option: 

Nice a simple!

Reply:
When we say perform any one of the following steps, it implies that these are options. No need to call it out as Option 1 and 2.

Reply:
> * Option 1: Through the `--set` command line option:

This is an incomplete sentence and there is no instruction of the step for the user to perform.
Suggest to write similar to this: "From the --set option in the Red Hat Helm chart-verifier CLI, set the configuration values of the chart-testing check as follows:"